package.json

```diff
@@ -49,7 +49,8 @@
     "commit-lint": "commitlint --edit -o",
     "semantic-release": "semantic-release",
     "generate:code-docs": "typedoc",
-    "pre-commit": "run-s check-types lint:fix audit format test generate:code-docs"
+    "pre-commit": "run-s check-types lint:fix audit format test generate:code-docs",
+    "example": "tsx -r dotenv/config"
```

Comment:
What is this meant to do? My read on that is it will run compile on all files?

Reply:
It runs all of the example scripts in the examples folder